lib/builder_web.ml

```diff
@@ -352,6 +352,39 @@ let routes ~datadir ~cachedir ~configdir ~expired_jobs =
     let job_name = Dream.param req "job" in
     let platform = Dream.query req "platform" in
     redirect_latest req ~job_name ~platform ~artifact:""
+
+  in
+
+  let return_latest_uuid req =
+    let job_name = Dream.param req "job" in
+    let platform = Dream.query req "platform" in
+    (Dream.sql req (Model.job_id job_name) >>= Model.not_found >>= fun job_id ->
+      Dream.sql req (Model.latest_successful_build_uuid job_id platform))
```

Comment:
there's some code duplication now -- should that be shared (between redirect_latest and this function here).